src/core/src/bootstrap/EnvLayer.py

```diff
@@ -55,6 +56,9 @@ def __init__(self, real_record_path=None, recorder_enabled=False, emulator_enabl
         self.file_system = self.FileSystem(recorder_enabled, emulator_enabled, self.__write_record, self.__read_record,
                                            emulator_root_path=os.path.dirname(self.__real_record_path))
 
+        # Constant paths
+        self.etc_environment_file_path = "/etc/environment"
```

Comment:
We try to be OS agnostic in the environment layer - everything that is not, gets passed in usually. No need to block the PR on this; just FYI.

Reply:
From what I have seen online, every Linux distro has its global environment variables stored here. But I understand your point and will keep it in mind in the future.